README.md

```diff
@@ -100,7 +105,7 @@ Which one of the following concepts does this best illustrate?
 What is the `ui-sref` directive, and how is it used?
 
 ```text
-Your answer goes here...
+It is used in <a> tag just like href in order to reference the source location for that link.
```

Comment:
But how is it different from using `href`? What happens when you click on an anchor tag that uses `ui-sref`?